src/components/facet-filter-controls/Filters.tsx

```diff
@@ -16,8 +16,8 @@ export default component$<{
 	return (
 		<div>
 			<div class="hidden lg:block">
-				{state.facetsWithValues.map((facet: FacetWithValues) => (
-					<div class="border-b border-gray-200 py-6">
+				{_facetsWithValues.map((facet: FacetWithValues) => (
```

Comment:
using facets from state here does not update filters on route change. using _facetWithValues here works but breaks the open/close

Reply:
👍 I will investigate. great job 😄